packages/network/src/v2/schemas/tableSchemas.ts

```diff
@@ -6,7 +6,7 @@ import { IStore } from "@latticexyz/store/types/ethers-contracts/IStore.sol/ISto
 
 // worldAddress:tableId => schema
 // TODO: add chain ID to namespace?
-const schemaCache: Partial<Record<`${string}:${string}`, Promise<TableSchema>>> = {};
+const schemaCache: Record<`${string}:${string}`, Promise<TableSchema>> = {};
```

Comment:
any reason this changed?

I tend to use `Partial<Record<...>>` for any object that I'm filling in over time, otherwise the types don't reflect that a key can sometimes not have a value

Reply:
yep agree this should be `Partial`!